### PR TITLE
Harvest Hero Origins and Emerald

### PR DESCRIPTION
**Pull request by Bombfuse**

### CODE REVIEW

File: content/news/024/index.md [hunk 1]
```diff
@@ -67,8 +67,53 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Harvest Hero Origins](https://store.steampowered.com/app/1651500/Harvest_Hero_Origins/)
+
+![hho_header](./hho_header.png)
+
+Harvest Hero Origins is an arcade wave defense game, built in rust on top
```
Comment (17cupsofcoffee) [inline content/news/024/index.md:74]:
```suggestion
Harvest Hero Origins is an arcade wave defense game, built in Rust on top
```

File: content/news/024/index.md [hunk 2]
```diff
@@ -67,8 +67,53 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+of [Emerald]. A [steam] page has recently been made and the game is
```
Comment (17cupsofcoffee) [inline content/news/024/index.md:75]:
```suggestion
of [Emerald]. A [Steam][hho-steam] page has recently been made and the game is
```

File: content/news/024/index.md [hunk 3]
```diff
@@ -67,8 +67,53 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+Battle the oncoming waves of enemies with a friend in local co-op, unlock
+new playable characters and skins, and make your way to the top of the
+leaderboard!
+
+Additionally, HHO will be at [PAXWest](https://west.paxsite.com/)
+this year, so come checkout our booth
+if you'll be there!
+
+- [Steam](https://store.steampowered.com/app/1651500/Harvest_Hero_Origins/)
```
Comment (17cupsofcoffee) [inline content/news/024/index.md:86]:
```suggestion
- [Steam][hho-steam]
```

File: content/news/024/index.md [hunk 4]
```diff
@@ -67,8 +67,53 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+- [Twitter](https://twitter.com/GemdropGames)
+- [Emerald]
+
+[steam]: https://store.steampowered.com/app/1651500/Harvest_Hero_Origins/
```
Comment (17cupsofcoffee) [inline content/news/024/index.md:90]:
```suggestion
[hho-steam]: https://store.steampowered.com/app/1651500/Harvest_Hero_Origins/
```

File: content/news/024/index.md [hunk 5]
```diff
@@ -67,8 +67,53 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+[Emerald]: https://github.com/Bombfuse/emerald
+
 ## Engine Updates
 
+### [Emerald]
+
+![logo](./em_logo.svg)
```
Comment (17cupsofcoffee) [inline content/news/024/index.md:97]:
```suggestion
![Emerald logo](./em_logo.svg)
```

File: content/news/024/index.md [hunk 6]
```diff
@@ -67,8 +67,53 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+In addition to being portable, [Emerald] aims to be easy to use
+while providing quality features like built-in physics and
+Aseprite integration.
+
+If any of this sounds good to you, and you'd like to be a part of it,
+[Emerald] welcomes all contributors to help make an extremely portable
+game engine!
+
+- [Github](https://github.com/Bombfuse/emerald)
+- [Discord]
```
Comment (17cupsofcoffee) [inline content/news/024/index.md:112]:
```suggestion
- [Discord][emerald-discord]
```

File: content/news/024/index.md [hunk 7]
```diff
@@ -67,8 +67,53 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+[Emerald]: https://github.com/Bombfuse/emerald
+[Discord]: https://discord.gg/NHsz38AhkD
```
Comment (17cupsofcoffee) [inline content/news/024/index.md:115]:
```suggestion
[emerald-discord]: https://discord.gg/NHsz38AhkD
```

**Review by 17cupsofcoffee — APPROVED** [review]
Looks good, just making a few tweaks (mainly to satisfy my own paranoia about link IDs clashing) :)

File: content/news/024/index.md [hunk 8]
```diff
@@ -67,8 +67,47 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+Harvest Hero Origins is an arcade wave defense game by [Gemdrop Games][gemdrop],
+built in Rust on top of [Emerald]. A [Steam][hho-steam] page has recently been
+made and the game is set to release sometime at the end of this summer.
+
+Battle the oncoming waves of enemies with a friend in local co-op, unlock
+new playable characters and skins, and make your way to the top of the
+leaderboard!
+
+Additionally, HHO will be at [PAXWest](https://west.paxsite.com/)
+this year, so come checkout our booth
```
Comment (17cupsofcoffee) [inline content/news/024/index.md:83]:
```suggestion
this year, so come check out their booth
```

**Comment by Bombfuse** [comment]
awesome, thanks so much!

**Pull request merged**